src/netius/test/base/poll.py

```
@@ -0,0 +1,523 @@
+#!/usr/bin/python
```

Pylint (4.0.7)

[refactor] Useless option value for '--disable', 'C0326' was removed from pylint, see https://github.com/pylint-dev/pylint/pull/3577.

(R0022)